stable/coredns/templates/NOTES.txt

```diff
@@ -11,10 +11,9 @@ It can be accessed using the below endpoint
   NOTE: It may take a few minutes for the LoadBalancer IP to be available.
         You can watch the status by running 'kubectl get svc -w {{ template "coredns.fullname" . }}'
 
-{{- if contains "UDPNTCP" .Values.serviceProtocol }}
   NOTE: Some cloud environments cannot create external loadbalancers with both "TCP" and "UDP" protocols.
-        So in case of error, use either "UDP" or "TCP" protocol by setting `serviceProtocol` parameter.
-{{- end }}
+        So in case of error, use either "DNS" (UDP) or "TLS/gRPC" (TCP) protocol by setting
```

Comment:
This change is going to make this "error" appear every time without providing a clear resolution. Should this even be an error message and under what conditions should it be triggered?

Reply:
Well, I'm not sure about the exact conditions but I'm sure this shouldn't prevent the users from using this chart?

Should this even be an error message?
I think the original author @shashidharatd should decide this. This ain't an error message IMHO, but something the users should take notice.

Under what conditions:
From the message above, I guess that the conditions should be:
- Users are using "some" cloud environment that doesn't support both UDP and TCP at the same time.
- Users set both UDP and TCP.

The original script seems to check only the second part, so it was overtriggering the message anyways.

Reply:
Originally, the warning message was there to notify users about non-working configuration in some cloud environment. This message no longer is required as from the code either UDP or TCP port is chosen and not both. So we could just get rid of the message. @w4-sjcho, can you please remove the message.